docs/change-log/index.md

```diff
@@ -16,6 +16,59 @@ The change log lists updates per release, open issues, limitations, and deprecat
 - [Limitations](#limitations)
 - [Deprecations and removed code](#deprecations-and-removed-code)
 
+## Upgrading KFP, Python, and Pydantic
+Upgrading these three MLRun dependencies spans several releases.  The upgrades are comprised of:
+- KFP: from 1.8 to 2.0. KFP has 2 components: the KFP service, and the KFP client code (which is used in both the MLRun service and various MLRun clients).
+- Pydantic: from version 1 to 2.
+- Python: from 3.9 to 3.11.
+
+This section describes the current guidelines. Specific changes are listed under the relevant versions.
+
+### KFP
+MLRun supports the KFP 2.x server, but workflows still require the KFP 1.8 syntax. Usage guidelines:
+- Client code, workflow code and syntax (DSL) is still the KFP 1.8 syntax. Working with the newer KFP 2.x syntax is not yet supported by MLRun.
+- Starting with v1.8.0, KFP is not pre-installed on images such as `mlrun/mlrun`. The image `mlrun/mlrun-kfp` includes KFP, but works only with Python 3.9.
+- You can install KFP manually (`pip install kfp~=1.8`), for example, to run KFP pipelines locally using the KFP 1.8 client, which requires Python 3.9.
```

Comment:
This instruction is different than what's written below (`pip install mlrun[kfp18]`). I actually think the other option is more correct since it adds MLRun packages that support kfp submission. Best to consult with @adamdelman or @alonmr on this.